programmers/문자열-내림차순.js

```diff
@@ -0,0 +1,10 @@
+function solution(s) {
+     
+    let answer;
+    
+    answer = s.split("");
+    answer.sort();
+    answer.reverse();
+    
+    return answer.join("");
```

Comment:
```suggestion
return s.split('').sort().reverse().join('');
```
이렇게 한번에 붙여서 쓸 수도 있답니다😁